CMSIS/NN/Tests/UnitTest/TestCases/TestData/requantize_s64/weights_data.h

```diff
@@ -16,8 +16,8 @@
  * limitations under the License.
  */
 
-// Generated by generate_test_data.py using TFL version 2.4.1 as reference.
+// Generated by generate_test_data.py using TFL version 2.6.0 as reference.
 #pragma once
 #include <stdint.h>
 
-const q7_t requantize_s64_weights[16] = {-127, 32, 32, 0, 95, 32, 95, -95, -42, -85, -127, -106, 0, -64, 0, 106};
+const q7_t requantize_s64_weights[16] = {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0};
```

Comment:
Maybe we could improve this or a add a new test case so we get a higher value of acc..

Reply:
Done